index.html

```diff
@@ -7,6 +7,10 @@
     <script class='remove'>
           var respecConfig = {
               specStatus:     "CR"
+            , crEnd: "2019-06-18"
```

Comment:
Where does the June 18th come from?
We have:
* June 13: Deadline for CR Comments
* June 20: PR Transition Request
* June 27: Publication of PR
*